### DISCUSSION TITLE
Welcome to duel-defense Discussions!

### DISCUSSION THREAD

**Discussion by d10sfan** [duel-defense/duel-defense#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by justfreesites** [comment]
I really hope you are still working on this game, it's giving me Red Invasion by Milkstone studios which was a great game on Xbox Live Indie Games. https://www.milkstonestudios.com/red-invasion/

**Reply by d10sfan** [reply]
> I really hope you are still working on this game, it's giving me Red Invasion by Milkstone studios which was a great game on Xbox Live Indie Games. https://www.milkstonestudios.com/red-invasion/

Thank you! I plan to come back to it sometime soon, been really busy lately, but I still enjoyed making this alot.

**Reply by d10sfan** [reply]
Also, if you have any ideas or contributions, would love to have them as well!
